.github/workflows/e2e.yml

```diff
@@ -190,7 +184,7 @@ jobs:
       - name: Generate certs
         run: tools/bin/mage dev:certificates
       - name: Restore initialized sql db
-        run: tools/bin/mage dev:dbStart dev:sqlRestore
+        run: tools/bin/mage dev:dbStart dev:SQLRestore dev:SQLCreateSeedDB
```

Comment:
Nit: If I'm not mistaken, `mage` is case-insensitive, but wouldn't it be better to keep it consistent with the rest of commands use it like `sqlRestore` and `sqlCreateSeedDb` ?